psalm.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<psalm
+    errorLevel="7"
```

Comment:
Can we make it smaller?

Reply:
See my note in the PR description:

> Psalm is still set to the DevKit default (level 7) in order to pass. This should be lowered and issues fixed in a future PR.

I tried to keep this to just the DevKit changes - I had to touch the code a little just to get Rector to pass, but I would like to do Psalm changes in a separate PR. Anything lower than 7 will require changes to **src/**.

Reply:
Okay, I got it.